src/assets/source/sass/components/_about.scss

```diff
@@ -0,0 +1,36 @@
+#about {
+  .card {
+    max-width: 1500px;
+    opacity: 0.9;
+    width: 90%;
+    margin: 7% 10% 0;
```

Comment:
using `%` for margins is not a good practice, please use `px` or other measure unit.